src/vs/workbench/browser/parts/editor/resourceViewer.ts

```diff
@@ -370,7 +370,7 @@ class InlineImageView {
 			dispose: () => combinedDisposable(disposables).dispose()
 		};
 
-		const cacheKey = descriptor.resource.toString();
+		const cacheKey = descriptor.resource.toString() + descriptor.etag;
```

Comment:
`${descriptor.resource.toString()}${descriptor.etag}`

Isn't this better ? Es6 String Literals

Reply:
@amarpathak what are you smoking?

Reply:
@amadare42 That’s right. I think we can add a symbol between these two string. 

`${descriptor.resource.toString()}:${descriptor.etag}`

Reply:
@hwhung0111 well, I think you linked wrong person, but I want to mention that es6 string templating will call toString automatically so your line can be shortened:
`${descriptor.resource}:${descriptor.etag}`

Reply:
This is is not the best solution. ES6 literals are not suited for this purpose but injecting dynamic data into strings.

Reply:
@amadare42  I am sorry about that. And thanks for telling me about es6 string. 
@KristofferTolboll2 If I add a colon in the string, is it better to use ES6 literals.

Reply:
@hwhung0111 ES6 template literals are most used, when there are more static content, in this case, there is only the `:` which is static, everything else is dynamic, therefore it will decrease readability

Reply:
@KristofferTolboll2 I get your point. But why I still found `${message}: [${err.toString()}` and `${this.scope}.${key}` in source code? These are similar to this issue.